src/YesSql.Core/Provider/IDbProviderOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using YesSql.Core.Services;
+
+namespace YesSql.Core.Provider
+{
+    public interface IDbProviderOptions
+    {
+        string ProviderName { get; set; }
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1165805/24572495/0a539896-162e-11e7-8b1c-915529d0b9d8.png)

First comment of the discussion. Yes, because this property is not necessary, this class isn't either. Even more we already have a class with all the necessary properties, including `TablePrefix`. It's called `Configuration`. So we just need to have all these extension methods and new projects use it instead of `IDbProviderOptions`.